aixtend/factories/dataset_factory.py

```diff
@@ -42,53 +49,54 @@ def _create_dataset_from_response(cls, response: dict) -> Dataset:
         Returns:
             Dataset: Coverted 'Dataset' object
         """
-        return Dataset(response['id'], response['name'], response['description'])
+        return Dataset(
+            id=response["id"],
+            name=response["name"],
+            description=response["description"],
+            function=Function.SPEECH_RECOGNITION,
```

Comment:
Is this constant value correct?
```Suggestion
            function=response["function"],
```

Reply:
I do not have the service to read and onboard datasets from the new backend. Once I have it, I will update it.